plugins/integrations/kubernetes-service/src/main/java/com/cloud/kubernetes/cluster/KubernetesClusterManagerImpl.java

```diff
@@ -1270,29 +1280,64 @@ private SecurityGroup getOrCreateSecurityGroupForAccount(Account owner) {
         return securityGroup;
     }
 
+    @Override
+    @ActionEvent(eventType = KubernetesClusterEventTypes.EVENT_KUBERNETES_CLUSTER_CREATE,
+            eventDescription = "creating Kubernetes cluster", async = true)
```

Comment:
```suggestion
    @ActionEvent(eventType = KubernetesClusterEventTypes.EVENT_KUBERNETES_CLUSTER_START,
            eventDescription = "starting Kubernetes cluster", async = true)
```

Reply:
@vishesh92 this is not needed as it is done deliberately to allow publishing `Completed` state event for `EVENT_KUBERNETES_CLUSTER_CREATE`. Argument passed for the method here is also `CreateKubernetesClusterCmd` so it will be used only when the cluster is created.